manifests/z/zadjii/HackerNewsforCommandPalette/0.0.3.0/zadjii.HackerNewsforCommandPalette.installer.yaml

```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: zadjii.HackerNewsforCommandPalette
+PackageVersion: 0.0.3.0
+Platform:
+- Windows.Universal
+- Windows.Desktop
+MinimumOSVersion: 10.0.17763.0
+InstallerType: msix
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.WindowsAppRuntime.1.6

```

Comment:
```suggestion
  - PackageIdentifier: Microsoft.WindowsAppRuntime.1.6
  - PackageIdentifier: Microsoft.WindowsTerminal
```

Reply:
Talked to @denelon offline and we're pretty confident this shouldn't be needed - it's not a Terminal extension, it's _{{something else entirely}}_, so I'm not sure why adding the Terminal as a dependency fixed it?

Reply:
No worries - it's a work in progress 🤫